packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -111,188 +78,139 @@ const propTypes = {
-      return null;
-    }
+  const signalWord = variant === 'custom' ? custom.signalWord : <FormattedMessage id={`Terra.notification.dialog.${variant}`} />;
 
-    const {
-      header,
-      title,
-      startMessage,
-      endMessage,
-      content,
-      acceptAction,
-      rejectAction,
-      variant,
-      customIcon,
-      isOpen,
-      buttonOrder,
-      emphasizedAction,
-      primaryAction,
-      secondaryAction,
-      message,
-      ...customProps
-    } = this.props;
-    if (process.env.NODE_ENV !== 'production' && acceptAction === undefined && primaryAction === undefined && rejectAction === undefined && secondaryAction === undefined) {
-      // eslint-disable-next-line no-console
-      console.warn('At least one of `acceptAction`,`primaryAction`,`rejectAction`,`secondaryAction` props must be provided for Notification dialog');
-    }
-
-    const defaultHeader = variant === variants.CUSTOM ? '' : <FormattedMessage id={`Terra.notification.dialog.${variant}`} />;
-    const theme = this.context;
-    const notificationDialogClassNames = classNames(cx(
-      'notification-dialog',
-      theme.className,
-    ),
-    customProps.className);
-
-    /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
-    return (
-      <AbstractModal
-        ariaLabel="Notification Dialog"
-        aria-labelledby="notification-dialog-header"
-        aria-describedby={title ? 'notification-dialog-title' : 'notification-dialog-header'}
-        role="alertdialog"
-        classNameModal={notificationDialogClassNames}
-        isOpen={this.props.isOpen}
-        onRequestClose={() => {}}
-        closeOnEsc={false}
-        closeOnOutsideClick={false}
-        zIndex="9000"
-        data-terra-notification-dialog
-      >
-        <FocusTrap focusTrapOptions={{ returnFocusOnDeactivate: true, clickOutsideDeactivates: false, escapeDeactivates: false }}>
-          <div className={cx('notification-dialog-inner-wrapper')}>
-            <div className={cx('notification-dialog-container')} tabIndex="0">
-              <div id="notification-dialog-header" className={cx('header-body')}>{header || defaultHeader}</div>
-              <div className={cx('notification-dialog-body')}>
-                {variant
-                  && <div className={cx('icon-container')}>{getIcon(variant, customIcon)}</div>}
-                <div className={cx('text-wrapper')}>
-                  {title
-                    && <div id="notification-dialog-title" className={cx('title')}>{title}</div>}
-                  {(startMessage || message)
-                    && <div className={cx('message')}>{(startMessage || message)}</div>}
-                  {content
-                    && <div>{content}</div>}
-                  {endMessage
-                    && <div className={cx('message')}>{endMessage}</div>}
+  /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+  return (
+    <AbstractModal
+      ariaLabel="Notification Dialog"
```

Comment:
We probably don't want to include this, but ariaLabel _is_ a required prop. Any reason not to just set the ariaLabel to the signalWord and remove the aria-labeledby?

Reply:
I pulled this forward from the original implementation: https://github.com/cerner/terra-framework/pull/1105/files/3a52fd7c396653559d11629f38ba2c507b8c2cb8#diff-cb62dc7ce0197bc569a9952450bb9fe5L246

I can update this to be the signalWord though.

Reply:
updated to use signalWord's translation string: https://github.com/cerner/terra-framework/pull/1105/commits/c45612d9b69bb2982f5ca4adbcad9abaf6a161f5#diff-cb62dc7ce0197bc569a9952450bb9fe5R154